test/multifusion.jl

```diff
@@ -23,6 +23,11 @@ Istr = TensorKitSectors.type_repr(I)
         @test one(C1) == leftone(M) == rightone(Mop)
         @test one(D1) == rightone(M) == leftone(Mop)
 
+        @test @constinferred(isone(C0))
+        @test isone(D0)
+        @test isone(C1) == isone(D1) == false
+        @test isone(M) == isone(Mop) == false
```

Comment:
I know this is not necessarily a big thing, but I don't like it too much to compare booleans to `false`. Could you maybe just write `@test !isone(C1) && !isone(D1) && ...`?